app/lib/common/providers/common_providers.dart

```diff
@@ -73,8 +75,9 @@ final _accountAvatarProvider =
   final account = ref.watch(accountProvider);
   final thumbSize = sdk.api.newThumbSize(48, 48);
   final avatar = await account.avatar(thumbSize);
-  if (avatar.data() != null) {
-    return MemoryImage(avatar.data()!.asTypedList());
+  final avatarData = avatar.data();
```

Comment:
```suggestion
  // Only call data() once as it will consume the value and any subsequent
  // call will come back with `null`.
  final avatarData = avatar.data();
```